src/functions.js

```diff
@@ -0,0 +1,31 @@
+const updateStorage = (newList) => {
+  newList.forEach((value, id) => {
+    value.index = id;
+  });
+  localStorage.setItem('todo', JSON.stringify(newList));
+};
+
+const ClearStorage = () => {
+  localStorage.clear();
+  window.location.reload();
+};
+const addTask = () => {
+  const prevStorage = JSON.parse(localStorage.getItem('todo'));
+  if (prevStorage && prevStorage.length) {
+    return prevStorage;
+  }
+  const data = [];
+  updateStorage(data);
+  return data;
+};
```

Comment:
- [ ] For this project use static array with objects where each object containing three keys - description [string], completed [bool], index: [number]. Then use function to populate them.
`const tasks = [ {...},{...}]`